nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/TestTreeAnnotationNotification.java

```diff
@@ -31,4 +45,46 @@
     protected Class<? extends CommentManager> getType() {
         return TreeCommentManager.class;
     }
+
+    @Test
+    @Deploy("org.nuxeo.ecm.platform.comment:OSGI-INF/notification-subscription-contrib.xml")
+    public void testAutoSubscribingOnlyOnceToNewAnnotations() {
```

Comment:
i am wondering if the `/nuxeo/nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/resources/OSGI-INF/comment-notification-contrib.xml` is not duplication of `nuxeo-platform-comment/src/main/resources/OSGI-INF/comment-notification-contrib.xml` as the both enable the auto-subscription

Reply:
Yes it is a re enable ->
Default behavior enables
CommentFeature disables
My test uses CommentFeature but needs to enable, so it re enables